routes/web.php

```diff
@@ -20,12 +21,14 @@
 Route::get('/Q&A', function () {
     return view('Q&A');
 })->name('Q&A');
-Route::middleware(['auth:sanctum', 'verified'])->get('/ask-question', function () {
-    return view('ask-question');
-})->name('ask-question');
 Route::middleware(['auth:sanctum', 'verified'])->get('/dashboard', function () {
     return view('dashboard');
 })->name('dashboard');
-
+Route::get('/Q&A', [QAController::class, 'guest']);
+Route::middleware(['auth:sanctum', 'verified'])->get('/deleteQuestion/{id}', [QAController::class, 'delete']);
+Route::middleware(['auth:sanctum', 'verified'])->get('/createQuestion', [QAController::class, 'create']);
+Route::middleware(['auth:sanctum', 'verified'])->get('/updateQuestion/{id}', [QAController::class, 'update']);
+Route::middleware(['auth:sanctum', 'verified'])->get('/ask-Question', [QAController::class, 'askQuestion']);
+Route::middleware(['auth:sanctum', 'verified'])->post('/submitQuestion', [QAController::class, 'submitQuestion']);
 Route::middleware(['auth:sanctum', 'verified'])->get('/planning', [PlanningController::class, 'index']);
 Route::middleware(['auth:sanctum', 'verified'])->get('/planning/appointment/{id}', [PlanningController::class, 'show']);
```

Comment:
Is misschien gelijk een goede kans om 'Route Groups' te gebruiken. zorgt ervoor dat er een groep bestaat voor de middleware die hier wordt gebruikt
https://laravel.com/docs/8.x/routing#route-groups